src/core/utils/CorePlugin.ts

```diff
@@ -1,13 +1,14 @@
 import { JWPlugin } from '../JWPlugin';
 import JWEditor from '../JWEditor';
-import { VRangeLocation, RangeDirection } from '../stores/VRange';
+import { VRangeLocation, RangeDirection, RelativePosition } from '../stores/VRange';
 
 export class CorePlugin extends JWPlugin {
     editor: JWEditor;
     handlers = {
         intents: {
             remove: 'onRemoveIntent', // names are just to show relationships here
             render: 'render',
+            selectAll: 'navigate',
```

Comment:
At some point, we'll need to stop and think about the `intentions` and `commands` we defined so far.